### PR TITLE
EWL-5810 global header (merge before PR #439  and #440)

### PR DESCRIPTION
**Pull request by designerneil**
## JIRA Ticket(s)
- [EWL-5810: A1 | Theme Global Header](https://issues.ama-assn.org/browse/EWL-5810)

## Description:
Finish theming the global header

## To Test:
- visit http://localhost:3000/?p=organisms-main-navigation-as-non-member
- click on the usermenu link that says `John Smithe`
- observe the page doesn't reload or jump to the top

## Relevant Screenshots/GIFs:
screen shot 2018-10-10 at 3 29 20 pm

## Additional Notes:
this PR is a dependency of https://github.com/AmericanMedicalAssociation/ama-d8/pull/875

### This PR is the originator of both #439 and #440. I branched off of this PR to make both of those PRs and needs to be merged before both of thos PRs
---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)

### CODE REVIEW

**Review by JaeBea — DISMISSED** [review]
@designerneil ,
This PR has work that doesn't seem related to the header. Could you please remove and clarify the work that this does in your description?

**Comment by designerneil** [comment]
This PR addresses all the styling issues in the global header.

This includes the user menu, search and logo

@JaeBea here's what changed:

- search submit in the header needed to be updated to match SG2
- The magnify icon needed to be white not blue -- a white icon was put into the icon library
- The category menu JS was causing causing error which in turn prevented the user header menus from firing propery
- The user menu drop down was not working properly. The JS needed to be updated
- The search input box did not match SG2 . Updates were made to correct the styling issues
- Please refer to PR https://github.com/AmericanMedicalAssociation/ama-d8/pull/875

If you test the D8 side without this PR you'll see that the header looks broken

**Comment by JaeBea** [comment]
@designerneil This looks like it has changes that I have seen in two other PRs. Please confirm if this PR is to replace:
- [EWL-5814 Themes global search #439](https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/439)
- [Updates search results for D8 #440](https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/440)

**Comment by designerneil** [comment]
@JaeBea This PR is the originator of both #439 and #440. I branched off of this PR to make both of those PRs

File: styleguide/source/assets/js/sign-in-dropdown.js [hunk 1]
```diff
@@ -3,16 +3,24 @@
     attach: function (context, settings) {
       var signInDropdown = $('.ama__sign-in-dropdown');
```
Comment (nstriedinger) [inline styleguide/source/assets/js/sign-in-dropdown.js:4]:
```suggestion
      var $signInDropdown = $('.ama__sign-in-dropdown');
```

Comment (nstriedinger) [reply styleguide/source/assets/js/sign-in-dropdown.js:4]: Drupal JS code standards: https://www.drupal.org/docs/develop/standards/javascript/javascript-coding-standards

**Review by nstriedinger — APPROVED** [review]
LGTM without the Drupal JS standards.

**Pull request merged**